### PR TITLE
Add support for generating docs

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: bootstrap.yml [hunk 1]
```diff
@@ -444,6 +444,21 @@ tools:
       - args: ['ninja', 'install']
         quiet: true
 
+  - name: host-xxd
```
Comment (Dennisbonke) [inline bootstrap.yml:447]: Missing labels and the architecture field.

File: bootstrap.yml [hunk 2]
```diff
@@ -686,6 +701,29 @@ tools:
     install:
       - args: ['ninja', 'install']
 
+  - name: host-hdoc
```
Comment (Dennisbonke) [inline bootstrap.yml:704]: Missing labels and the architecture field.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Two items that will break xbbs otherwise. Outside that, LGTM

**Comment by no92** [comment]
Addressed the issues raised.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**